game/client_types/logic.js

```diff
@@ -168,13 +169,13 @@ module.exports = function(treatmentName, settings, stager, setup, gameRoom) {
     stager.extendStep('results', {
         cb: function() {
 
-            var results, previousStage;
+            //var results, previousStage;
 
-            previousStage = node.game.plot.previous(
+            let previousStage = node.game.plot.previous(
                 node.game.getCurrentGameStage()
             );
```

Comment:
This can be simplified with: `let previousStep = node.game.getPreviousStep();`
Note that I changed the name stage to step, because it is the previous step (please adapt the code below as well).